components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/api/impl/UserImpl.java

```diff
@@ -195,31 +194,23 @@ public Response updateUser(UserWrapper userWrapper, @QueryParam("username") Stri
                             new String(decodedBytes, "UTF-8"));
```

Comment:
Better to get the "UTF-8" from [1].

[1]. http://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html